src/stores/userPreferences.ts

```diff
@@ -0,0 +1,32 @@
+import { writable } from "svelte/store";
+
+type UserPreferences = {
+  broken_experimental_features: boolean;
+  last_sync_timestamp: number;
+  sync_to_single_page: boolean;
+};
```

Comment:
We've somehow ended up with two, maybe three ways of getting "user preferences" / "settings". I think it's worth tidying that up.

I think two ways is probably fine, but the split should be obvious, _and_ this Svelte wrapper should really use the LogseqServiceWrapper instead of calling logseq's SDK directly.